installscripts/cookbooks/jenkins/recipes/configurejenkins.rb

```diff
@@ -92,84 +152,171 @@
+  command "#{node['script_root']}/cognitouser.sh #{node['jenkins']['clicommand']} #{node['cognitouser']} #{node['cognitopassword']}"
 end
 
-execute 'configJenkinsLocConfigXml' do
-  command "#{node['cookbook_root']}/jenkins/files/node/configJenkinsLocConfigXml.sh  #{node['jenkinselb']} #{node['jenkins']['SES-defaultSuffix']}"
+#Set up create-service job
+cookbook_file "#{node['script_root']}/job_create-service.sh" do
+  source 'jobs/job_create-service.sh'
+  mode 0755
 end
 
 execute 'createJob-create-service' do
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_create-service.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmpath']}"
+  command "#{node['script_root']}/job_create-service.sh #{node['jenkins']['clicommand']} #{node['scmpath']}"
+end
+
+#Set up delete-service job
+cookbook_file "#{node['script_root']}/job_delete-service.sh" do
+  source 'jobs/job_delete-service.sh'
+  mode 0755
 end
 
 execute 'createJob-delete-service' do
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_delete-service.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmpath']}"
+  command "#{node['script_root']}/job_delete-service.sh #{node['jenkins']['clicommand']} #{node['scmpath']}"
+end
+
+#Set up build_pack_api job
+cookbook_file "#{node['script_root']}/job_build_pack_api.sh" do
+  source 'jobs/job_build_pack_api.sh'
+  mode 0755
 end
 
 execute 'createJob-job_build_pack_api' do
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_build_pack_api.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmpath']}"
+  command "#{node['script_root']}/job_build_pack_api.sh #{node['jenkins']['clicommand']} #{node['scmpath']}"
 end
 
-execute 'createJob-bitbucketteam_newService' do
-  only_if { node['scm'] == 'bitbucket' }
-  command "#{node['cookbook_root']}/jenkins/files/jobs/job_bitbucketteam_newService.sh #{node['jenkinselb']} #{node['jenkins']['clientjar']} #{node['authfile']} #{node['scmelb']}"
+if node['scm'] == 'bitbucket'
+  #Set up bitbucketteam_newService job
+  cookbook_file "#{node['script_root']}/job_bitbucketteam_newService.sh" do
+    source 'jobs/job_bitbucketteam_newService.sh'
+    mode 0755
+  end
+
+  execute 'createJob-bitbucketteam_newService' do
+    command "#{node['script_root']}/job_bitbucketteam_newService.sh #{node['jenkins']['clicommand']} #{node['scmelb']}"
+  end
+
+  #Set up platform_api_services job
+  cookbook_file "#{node['script_root']}/job_platform_api_services.sh" do
+    source 'jobs/job_platform_api_services.sh'
+    mode 0755
+  end
+
+  execute 'createJob-platform_api_services_bitbucket' do
+    command "#{node['script_root']}/job_platform_api_services.sh #{node['jenkins']['clicommand']} #{node['scmelb']}"
+  end
+end
+
+#Set up platform_api_services job
```

Comment:
@bleggett Code duplicate. Its already defined above. #Set up platform_api_services job

Reply:
It is defined once in a 
```
if node['scm'] == 'bitbucket'
```
and once outside of it, because the original code had a conditional there as well.

If you're saying that it should now be `scmelb` in both cases and so there is no need to conditionalize that step, will remove it.